### PR TITLE
Applying Upstream Updates: add Terminus commands

### LINKED ISSUE #1171: Applying Upstream Updates Doc Update
Re: [Applying Upstream Updates](https://pantheon.io/docs/articles/sites/code/applying-upstream-updates)

## Issue Description:

Doesn't cover terminus

## Suggested Resolution 

Mention that upstream updates can be applied via Terminus. Show how to learn more about terminus commands related to applying upstream updates:

- terminus help site upstream-updates
- terminus help sites mass-update

### PR DESCRIPTION
**Pull request by nataliejeremy**
Closes #1171 
Added commands for applying upstream updates via Terminus. @ari-gold Please review--thanks!

### CODE REVIEW

File: source/docs/articles/sites/code/applying-upstream-updates.md [hunk 1]
```diff
@@ -22,8 +22,27 @@ Only use the one-click updates on the Dashboard to update your site's core. Do n
...
+
+If you prefer using the command line, you can apply updates with [Terminus](/docs/articles/local/cli/).
+
+### Update a Specific Site
+
+```
+terminus site upstream-updates apply --site=site --env=env --accept-upstream --updatedb
+```
+
+Learn more about this command by running `terminus help sites upstream-updates`.
```
Comment (ari-gold) [inline source/docs/articles/sites/code/applying-upstream-updates.md]: site should be singular
sites >> site

File: source/docs/articles/sites/code/applying-upstream-updates.md [hunk 2]
```diff
@@ -22,8 +22,27 @@ Only use the one-click updates on the Dashboard to update your site's core. Do n
...
 ![The Code tab in the Pantheon site's dashboard's Live Environment, showing the upstream commits in the log as deployed.](/source/docs/assets/images/desk_images/357435.png)
 
+## Apply Upstream via Terminus
+
+If you prefer using the command line, you can apply updates with [Terminus](/docs/articles/local/cli/).
+
+### Update a Specific Site
+
+```
+terminus site upstream-updates apply --site=site --env=env --accept-upstream --updatedb
```
Comment (ari-gold) [inline source/docs/articles/sites/code/applying-upstream-updates.md]: --site=<site>  --env=<env>

Why is --accept-upstream necessary (add reason)?

File: source/docs/articles/sites/code/applying-upstream-updates.md [hunk 3]
```diff
@@ -22,8 +22,27 @@ Only use the one-click updates on the Dashboard to update your site's core. Do n
 8. Click **Visit Live Environment** to verify the update is live.
 ![The Code tab in the Pantheon site's dashboard's Live Environment, showing the upstream commits in the log as deployed.](/source/docs/assets/images/desk_images/357435.png)
 
+## Apply Upstream via Terminus
```
Comment (ari-gold) [inline source/docs/articles/sites/code/applying-upstream-updates.md]: Add "Updates"
>> Apply Upstream Updates via Terminus

File: source/docs/articles/sites/code/applying-upstream-updates.md (hunk 2 above)
Comment (nataliejeremy) [inline source/docs/articles/sites/code/applying-upstream-updates.md]: @ari-gold When I ran `terminus help site upstream-updates` it listed that as an option to automatically resolve conflicts. Should I remove it? And do you think we really need to explain each option or let the user run the help command to see the descriptions?

Comment (ari-gold) [inline source/docs/articles/sites/code/applying-upstream-updates.md]: I like pointing them to the command for help. Maybe leave out the `--accept-upstream` option and someone can find that option in the help?

Comment (nataliejeremy) [inline source/docs/articles/sites/code/applying-upstream-updates.md]: +1 I made the changes to the doc. Thanks @ari-gold

**Pull request merged**